packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts

```diff
@@ -76,6 +77,22 @@ export interface LoadBalancedFargateServiceProps {
    * @default false
    */
   publicTasks?: boolean;
+
+  /**
+   * Domain name for the service, e.g. api.example.com
+   */
+  domainName?: string;
+
+  /**
+   * Route53 hosted zone name for the domain, e.g. "example.com."
+   */
+  domainZone?: string;
+
+  /**
+   * ARN of the Certificate Manager SSL certificate to associate with the load balancer.
+   * Setting this option will set the load balancer port to 443.
+   */
+  certificateArn?: string;
```

Comment:
Use a [CertificateRef](https://awslabs.github.io/aws-cdk/refs/_aws-cdk_aws-certificatemanager.html#certificateref) instead of a string ARN.

Reply:
Done